README.md

```diff
@@ -27,4 +27,10 @@ ACAS-Roo Server
 ### Who do I talk to? ###
 
 * Repo owner or admin
-* Other community or team contact
+* Other community or team contact
+
+
+## JChem Changes
+
+* For JChem and Java 11, there are new changes that use the JChem Maven repository for the build process
+* This requires additional authentication changes that reference ~/.m2/settings.xml

```

Comment:
I think we need more detail. Can we provide a settings.xml for use here? Wondering also if it's possible to keep the old build process.  I know we have a few automated builds on servers which could benefit from using the old big fat jar method.  i.e. can we make the Dockerfile-jchem file and Dockerfile-multistage file still use a jchem jar if it's provided rather than the settings.xml?